README.md

```diff
@@ -49,10 +57,11 @@ Using Ruby, remove Belle from the town residents, and
 add her to the list of guests in the castle.
 
 Write your code here:
-```ruby
-# code here
-```
-
+residents.remove ("Belle")
+def add (guest)
+    @guests << guests
+    end
+    residents.add("Belle")
```

Comment:
You have some of the correct pieces here, but you are not pushing into the correct array.

`@guests` does not exist. Instead, you need to drill into the hash again and make sure you're adding to the correct array.

```rb
town[:castle][:guests] << "Belle"
```